tests/eosio.msig_tests.cpp

```diff
@@ -329,6 +331,20 @@ BOOST_FIXTURE_TEST_CASE( propose_with_wrong_requested_auth, eosio_msig_tester )
 
 
 BOOST_FIXTURE_TEST_CASE( big_transaction, eosio_msig_tester ) try {
+   //change `default_max_inline_action_size` to 512 KB
+   create_account(N(eosio.bios));
+   set_code(N(eosio.bios), contracts::bios_wasm());
+   set_abi(N(eosio.bios), contracts::bios_abi().data());
```

Comment:
There should be no need to create the account `eosio.bios` and deploy the bios contract to it. The bios contract should already be deployed to the `eosio` account at the start of this test. That is why the `setparams` action sent to `config::system_account_name` (aka `eosio`) works below.